### PR TITLE
Cosmos DB | Update for Az.CosmosDB 0.1.4 | Common

### PR DESCRIPTION
**Pull request by plzm**
## Description

For Az.CosmosDB 0.1.4 release, update samples for account/common-focused cmdlets.

## Checklist

- [ ] This pull request was tested on __both of__:
  - [X] PowerShell 5.1 (Windows)
  - [ ] PowerShell 6.x
  - [X] PowerShell 7.x ([Latest PowerShell](https://github.com/PowerShell/PowerShell/releases))
- [X] Scripts do not contain static passwords or other secret tokens.
- [X] All Azure resource identifiers which must be universally unique are guaranteed to be so.

### Testing information

Windows 10.0.18363, Powershell 5.1, Az 3.8.0
Windows 10.0.18363, Powershell 7.0.0, Az 3.8.0

### CODE REVIEW

**Comment by plzm** [comment]
#sign-off

File: cosmosdb/common/ps-account-update-region.ps1 [hunk 1]
```diff
@@ -11,28 +11,18 @@
...
+
+ForEach ($location in $locations) {
+    $locationObjects += New-AzCosmosDBLocationObject -LocationName $location -FailoverPriority ($i++)
 }
 
-Set-AzResource -ResourceGroupName $resourceGroupName -ResourceType $resourceType `
-    -ApiVersion $apiVersion -Name $accountName `
-    -PropertyObject $CosmosDBProperties -Force
+Write-Host "Update Cosmos DB account region(s)"
+Update-AzCosmosDBAccountRegion -InputObject $account -LocationObject $locationObjects
```
Comment (markjbrown) [inline cosmosdb/common/ps-account-update-region.ps1:28]: Can a user update a region and specify it is AZ enabled?

Comment (plzm) [reply cosmosdb/common/ps-account-update-region.ps1:28]: Yes, assuming the region object-based approach is used as in ps-account-update-region.ps1. 

However, there is not at this time a convenient way to look up programmatically if a region is AZ-enabled; user must consult https://azure.microsoft.com/global-infrastructure/regions/ or similar resource to determine whether the region(s) in question is/are AZ-enabled (there is feedback about this on docs and github in various places, e.g. https://github.com/MicrosoftDocs/feedback/issues/544).

Once user determines if a region is AZ-enabled, then would need to so denote by adding the -IsZoneRedundant $true/$false parameter to New-AzCosmosDBLocationObject.

I'll extend the sample to use a hashtable of (region, AZ-enabled) and build the location list that way to show the approach.

Comment (plzm) [reply cosmosdb/common/ps-account-update-region.ps1:28]: There is also this approach: run the following Powershell and look for any SKU showing enumerated AZs. This region - westus - does not show as AZ-enabled at this time, and this is borne out by none of the returned SKUs having an AZ list.
Get-AzComputeResourceSku | where {$_.Locations.Contains("westus")}

Comment (plzm) [reply cosmosdb/common/ps-account-update-region.ps1:35]: OK, pushed the extended sample with AZ designation.

**Pull request merged**